website/meta/sidebars.json

```diff
@@ -36,7 +36,8 @@
                     },
                     { "text": "spaCy Projects", "url": "/usage/projects", "tag": "new" },
                     { "text": "Saving & Loading", "url": "/usage/saving-loading" },
-                    { "text": "Visualizers", "url": "/usage/visualizers" }
+                    { "text": "Visualizers", "url": "/usage/visualizers" },
+                    { "text": "Large Language Models", "url": "/usage/large-language-models", "tag": "new"  }
```

Comment:
If @ines agrees,  perhaps we can move it up a bit and have "Large Language Models" just below "Training models". I would also consider removing the "new" annotations from all the other sections (that are not really new anymore) to allow focusing more on the LLM subsection.

Reply:
Should I do this or wait for Ines to chime in?

Reply:
After internal discussion: you can move the new LLM section to below "Embeddings & Transformers", and remove the "new" annotations from all other sections :-)